frontend/components/Header.vue

```diff
@@ -7,10 +7,57 @@
         class="logo"
       />
     </a>
-    <button class="login-button">Login</button>
+    <button v-if="loggedIn" class="logout-button" @click="logout">Logout</button>
```

Comment:
I dont think we should use a "Logout" Button for the Header. Rather a button with a profile logo/image which leads to a profile side where you can logout